docs/css-classes-reference.rst

```diff
@@ -605,6 +605,14 @@ AspectJ ("aspectj")
 * ``inheritance``:      keywords "extends" and "implements" inside an aspect header
 * ``title``:            aspect, (intertype) method name or pointcut name inside an aspect header
 
+Fortran ("fortran", "f77", "f90", "f95")
```

Comment:
Yes. Having both in the same plugin would probably make the syntax file much more complicated. We could probably remove the ``f77`` and keep only ``("fortran", "f90", "f95")``, and leave fixed-form syntax for another f77 definition file.